src/com/lftechnology/java/training/alina/jdbc/service/InputValidation.java

```diff
@@ -0,0 +1,78 @@
+package com.lftechnology.java.training.alina.jdbc.service;
+
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+
+/**
+ * InputValidation consists of validations regarding user input data
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class InputValidation {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)